gwcs/tags/tests/test_selector.py

```diff
@@ -0,0 +1,44 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, unicode_literals, print_function
+
+import numpy as np
+
+from astropy.modeling import models as astmodels
+from astropy.tests.helper import pytest
+from gwcs import selector
+from pyasdf.tests import helpers
+
+
+
+def test_regions_selector(tmpdir):
+    m1 = astmodels.Mapping([0, 1, 1]) | astmodels.Shift(1) & astmodels.Shift(2) & astmodels.Shift(3)
+    m2 = astmodels.Mapping([0, 1, 1]) | astmodels.Scale(2) & astmodels.Scale(3) & astmodels.Scale(3)
```

Comment:
I feel like this would be more readable if this just imported the classes directly into the namespace:

`from astropy.modeling.models import Mapping, Shift, etc...`